src/Protocol/Protocol110x.cpp

```diff
@@ -0,0 +1,879 @@
+
+// Protocol110x.cpp
+
+/*
+Implements the 1.10.x protocol classes:
+	- cProtocol1100
+		- release 1.10.0 protocol (#210)
+(others may be added later in the future for the 1.10 release series)
+*/
+
+#include "Globals.h"
+#include "Protocol110x.h"
+#include "Packetizer.h"
+
+#include "../Root.h"
+#include "../Server.h"
+
+#include "../Entities/Boat.h"
+#include "../Entities/ExpOrb.h"
+#include "../Entities/Minecart.h"
+#include "../Entities/FallingBlock.h"
+#include "../Entities/Painting.h"
+#include "../Entities/Pickup.h"
+#include "../Entities/Player.h"
+#include "../Entities/ItemFrame.h"
+#include "../Entities/ArrowEntity.h"
+#include "../Entities/FireworkEntity.h"
+#include "../Entities/SplashPotionEntity.h"
+
+#include "../Mobs/IncludeAllMonsters.h"
+
+#include "Bindings/PluginManager.h"
+
+
+
+
+
+// The disabled error is intended, since the Metadata have overlapping indexes
+// based on the type of the Entity.
+//
+// IMPORTANT: The enum is used to automate the sequential counting of the
+// Metadata indexes. Adding a new enum value causes the following values to
+// increase their index. Therefore the ordering of the enum values is VERY important!
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wduplicate-enum"
```

Comment:
These two pragmas trigger a warning in MSVC, they should have been in an `#ifdef` block